content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_addon.md

```diff
@@ -0,0 +1,93 @@
+
+安装所需插件（addons）以通过一致性测试
+<!--
+Installs required addons for passing Conformance tests
+-->
+
+### Synopsis
+
+此命令不能单独运行。 请参阅可用的子命令列表。
+<!--
+This command is not meant to be run on its own. See list of available subcommands.
+-->
+
+### 可选项
+
+<table style="width: 100%; table-layout: fixed;">
+  <colgroup>
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">addon 帮助。</td>
+    </tr>
+
+  </tbody>
+</table>
+
+<!--
+<table style="width: 100%; table-layout: fixed;">
+  <colgroup>
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">help for addon</td>
+    </tr>
+
+  </tbody>
+</table>
+
+-->
+
+
+
+### 从父命令继承的可选项
+
+<table style="width: 100%; table-layout: fixed;">
+  <colgroup>
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">[实验] 到'真实'主机根文件系统的路径。</td>
+    </tr>
+
+  </tbody>
+</table>
+
+<!--
+<table style="width: 100%; table-layout: fixed;">
+  <colgroup>
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">[EXPERIMENTAL] The path to the 'real' host root filesystem.</td>
```

Comment:
better with line-by-line translation, easy to compare?

Reply:
Sure.